lang/en.json

```
@@ -669,6 +671,7 @@
     "UsesIntervalTurn": "Turn",
     "VehicleCrew": "Crew",
     "VehicleDialogSwap": "Are you trying to swap a driver and a passenger?",
+    "VehicleDialogSwapTitle": "Flip Driver and Passenger?",
```

```suggestion
    "VehicleDialogSwapTitle": "Swap Driver and Passenger?",
```
Just so it's using a consistent term